charts/index.yaml

```diff
@@ -262,6 +262,15 @@ entries:
     urls:
     - https://raw.githubusercontent.com/avoltz/blob-csi-driver/staging/charts/v4.3.0/blob-csi-driver-v4.3.0.tgz
     version: v4.3.0
+  - apiVersion: v1
+    appVersion: v4.4.0
+    created: "2023-10-12T19:27:00.820283000Z"
+    description: Azure Blob Storage CSI driver
+    digest: 86a1f17a630f37d818c4cb1dd69e6c1711c64c411de6f808bb7a006a40097b71
+    name: blob-csi-driver
+    urls:
+    - https://raw.githubusercontent.com/avoltz/blob-csi-driver/staging/charts/v4.4.0/blob-csi-driver-v4.4.0.tgz
+    version: v4.4.0
```

Comment:
there should be a file called `BLOB_CSI_VERSION` in the repo

could you update this file to point to the new version?

Reply:
Yes. I have a PR for that

Reply:
where is that pr?

Reply:
i think it would make the most sense to include it in this pr, that way the automation will pick it up and create a pr in the ado clone to merge with the new code in

Reply:
> there should be a file called `BLOB_CSI_VERSION` in the repo
> 
> could you update this file to point to the new version?

Version bump PR: https://msazure.visualstudio.com/One/_git/Hydra/pullrequest/8913392